src/common/function/src/function.rs

```diff
@@ -15,21 +15,22 @@
 use std::fmt;
 use std::sync::Arc;
 
-use chrono_tz::Tz;
 use common_query::error::Result;
 use common_query::prelude::Signature;
+use common_time::timezone::get_timezone;
+use common_time::Timezone;
 use datatypes::data_type::ConcreteDataType;
 use datatypes::vectors::VectorRef;
 
 #[derive(Clone)]
 pub struct FunctionContext {
-    pub tz: Tz,
+    pub timezone: Timezone,
```

Comment:
Is it possible to access `QueryContext` from UDF?

Reply:
@waynexia already has a solution, but no time to implement 
![image](https://github.com/GreptimeTeam/greptimedb/assets/14142/e5e47aaf-5354-446b-8b73-5a01c39190d0)

Reply:
Discussed with datafusion developer, the implementation of built-in `now` can be referenced: https://github.com/apache/arrow-datafusion/blob/main/datafusion/physical-expr/src/functions.rs#L499

Reply:
After discussing with @waynexia , we have a temporal solution. Finally, we hope that Datafusion could change the API that can accept a context for functions while evaluating.